src/trace/tracer-wrapper.ts

```diff
@@ -99,4 +116,23 @@ export class TracerWrapper {
     this.tracer.inject(span, "text_map", dest);
     return dest;
   }
+
+  public setConsumeCheckpoint(contextJson: any, eventType: string, arn: string): void {
+    if (!arn) {
+      logDebug("DSM: No ARN provided, skipping setConsumeCheckpoint");
+      return;
+    }
+
+    if (!this.config.dataStreamsEnabled) {
+      return;
+    }
```

Comment:
Can we put this before the arn check, that way when dsm is disabled, users will never see dsm related messages in their logs.